src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceRepository.kt

```diff
@@ -22,8 +22,10 @@ interface LicenceRepository : JpaRepository<Licence, Long>, JpaSpecificationExec
         JOIN l.submittedBy com
         WHERE (l.actualReleaseDate = CURRENT_DATE OR l.conditionalReleaseDate = CURRENT_DATE) 
         AND l.statusCode = 'SUBMITTED'
-        AND EXISTS 
-                (SELECT 1 FROM AuditEvent ae WHERE l.id = ae.licenceId AND ae.detail LIKE '%APPROVED%')
+        AND (
+            (EXISTS (SELECT 1 FROM AuditEvent ae WHERE l.id = ae.licenceId AND ae.detail LIKE '%APPROVED%')) 
+         OR (l.versionOfId IS NOT NULL) 
+        )
     """,
```

Comment:
This query finds in progress licences that were previously approved. Before versioning any in progress licence that was previously approved will have an approved audit event.  After versioning is enabled in progress version won't have an approved audit event but will have a versionOfId value.

Reply:
Am I right in thinking this won't select licences that are `IN_PROGRESS` (even if they're edits)? Is that intentional?

Reply:
ahh, I see my comment above mentioned IN_PROGRESS when this only looks at SUBMITTED licences :) Yes you are right that this only selects licences that have been previously approved and now have a status of SUBMITTED. it doesn't return any IN_PROGRESS licences that have been previously approved. It behaved like this before so I assume that was correct behaviour, but I'm now thinking it might not be.

Reply:
I don't think this was right before, going to check with Laura what it should be doing

Reply:
ok, discussed with Laura and it was intended to only send notifications for SUBMITTED licences is the correct behaviour.